content/docs/latest/spire-helm-charts-hardened-about/recommendations.md

```diff
@@ -0,0 +1,89 @@
+to disable them.
+
+your-values.yaml snippet:
+```yaml
+global:
+  spire:
+    recommendations:
+      enabled: true
+```
+
+# Individual Recommentations
+
+| Name                                                                                                                                | Value                                          |
+| ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------- |
+| [Namespace Layout](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-layout)                                 | global.spire.recommendations.namespaceLayout   |
+| [Namespace Pod Security Standards](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-pod-security-standards) | global.spire.recommendations.namespacePSS      |
+| [PriorityClassName](/docs/latest/spire-helm-charts-hardened-about/recommendations/#priority-class-name)                             | global.spire.recommendations.priorityClassName |
+| [Prometheus](/docs/latest/spire-helm-charts-hardened-about/recommendations/#prometheus)                                             | global.spire.recommendations.prometheus        |
+| [Strict Mode](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-layout)                                      | global.spire.recommendations.strictMode        |
+| [Security Contexts](/docs/latest/spire-helm-charts-hardened-about/recommendations/#security-contexts)                               | global.spire.recommendations.securityContexts  |
+
+## Namespace Layout
+
+Option `global.spire.recommendations.namespaceLayout` causes the services to be deployed
+across the two recommended namespaces for the services:
+
+| Namespace Type | Namespace Value                     | Default Value | Purpose                                                    |
+| -------------- | ----------------------------------- | ------------- | ---------------------------------------------------------- |
+| Server         | global.spire.namespaces.server.name | spire-server  | Services that should have restricted Kubernetes privileges |
+| System         | global.spire.namespaces.system.name | spire-system  | Services needing Kubernetes privileges                     |
+
+| Service Name                   | Namespace Type |
+| ------------------------------ | -------------- |
+| SPIRE Server                   | Server         |
+| SPIFFE OIDC Discovery Provider | Server         |
+| SPIFFE CSI Driver              | System         |
+| SPIRE Agent                    | System         |
+
+## Namespace Pod Security Standards
+
+Option `global.spire.recommendations.namespacePSS` sets the chart to set the recommended
+[Kubernetes Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/) labels when namespaces
+are created with the chart via:
```

Comment:
I think we need to reword this a bit to link to the namespaces settings

Reply:
Good call. Missed that part. I just took a stab at fixing it. Please have a look.